api/src/data/utils.clj

```diff
@@ -7,3 +7,8 @@
 
 (defn ts-now []
   (int (.getEpochSecond (Instant/now))))
+
+(defn set-dev-aws-system-properties []
```

Comment:
Move this to some test namespace. The idea is that is should be only used in tests.